master/docs/relnotes/0.9.0b2.rst

```diff
@@ -0,0 +1,89 @@
+Release Notes for Buildbot 0.9.0b2
+==================================
+
+..
+    Any change that adds a feature or fixes a bug should have an entry here.
+    Most simply need an additional bulleted list item, but more significant
```

Comment:
I don't get that sentence, may be
The most simple changes can be described with a bullet list. When more information is necessary, such changes
deserve to have their own subsection.